### DISCUSSION TITLE
Shelly PlugS

### DISCUSSION THREAD

**Discussion by CM-UH** [RavenSystem/esp-homekit-devices#1663] [MEPLHAA Scripts]
Hi,
did anyone use the power measurement of the Shelly PlugS ??
I have no clue how to do this?!?!

**Comment by RavenSystem** [comment]
I have included it in Devices-database.

**Reply by CM-UH** [reply]
Thanks a lot

**Reply by CM-UH** [reply]
I did some tests here are my tested config (with Chip type =1):
{"c":{"l":2,"b":[{"g":13,"t":5}]},
"a":[{"t":2,
"0":{"r":[{"g":15},{"g":0,"v":1}]},
"1":{"r":[{"g":15,"v":1},{"g":0}]},"b":[{"g":13}],
"es":[{"t":75,"n":1,"dt":[5,14,12],"vf":25.3,"cf":0.000078,"pf":1.845164}]}]}
